templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes" class="btn btn-danger">
```

Comment:
Cancel button will be more user friendly in this case:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on.

Reply:
Wasn't in readme.